docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -2,56 +2,57 @@
 Creating a Sawtooth Network
 ***************************
 
-This procedure describes how to create a Sawtooth network with two or more
+This procedure describes how to create a Sawtooth network with multiple
 validator nodes. It creates an application development environment that is
 similar to the single-node environment in :doc:`installing_sawtooth`, but with
 a different consensus mechanism to support additional nodes.
 
-* Docker: You will create a new application development environment by using a
-  Docker Compose file that creates a network with five validator nodes.
+You can install and run a multiple-node Sawtooth application development
+environment on one of the following platforms:
 
-* Ubuntu: You will add a node to the existing application development
-  environment that is described in :doc:`ubuntu`, but you will delete all
-  existing blockchain data, including the genesis block.
+* Docker: Run a Sawtooth network from prebuilt
+  `Docker <https://www.docker.com/>`_ containers.
 
-The following figure shows an example network with two validator nodes:
+- Ubuntu: Install Sawtooth natively using
+  `Ubuntu 16.04 <https://www.ubuntu.com/>`_. You will add a node to the existing
+  application development environment that is described in :doc:`ubuntu`, but
+  you will delete all existing blockchain data, including the genesis block.
 
```

Comment:
Sphinx isn't rendering a line between the Ubuntu and Kubernetes sections

Reply:
This is a Sphinx bug; I don't have any control over the spacing.  Unfortunately, Sphinx does this a lot. See the next bullet list for another example.